apps/settings/js/panels/operator_settings/views/roaming_preference_selector.js

```diff
@@ -0,0 +1,39 @@
+define(function(require) {
+  'use strict';
+
+  var Module = require('modules/base/module');
+
+  var RoamingPreferenceSelector =
+    Module.create(function RoamingPreferenceSelector(root) {
+      this._root = root;
+      this._manager = null;
+      
+      this._root.addEventListener('blur', () => {
+        this._manager.setRoamingPreference(this._root.value);
+      });
+      this._boundUpdate = this._update.bind(this);
+  });
+
+  RoamingPreferenceSelector.prototype._update = function(preference) {
```

Comment:
`update` is too vague to understand, can we add the noun for it to make it more specific ?